templates/telecommunications/index.html

```diff
@@ -0,0 +1,207 @@
+{% extends "templates/one-column.html" %}
+
+{% block meta_description %}Companies around the world rely on Ubuntu for secure open source solutions. We work with our customers to meet the highest security standards.{% endblock %}
+
+{% block title %}Ubuntu solutions for telecommunications{% endblock %}
+
+{% block content %}
+<section class="p-strip--accent is-deep">
+  <div class="row">
+    <div class="col-8">
+      <h1>Ubuntu for telcos</h1>
+      <p>As an established leader in cloud computing, Canonical let&rsquo;s you apply the webscale savings and best practice  that are necessary to succeed in the telco world. By using established open source software to automate the deployment and operation of NFVI and IoT solutions, we can help you become even more agile while reducing your costs and time to market.</p>
+      <p><a class="p-button--positive p-link--external" href="https://www.brighttalk.com/webcast/6793/290067" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'External Link', 'eventAction' : 'Watch Cloud to Edge webinar', 'eventLabel' : 'Watch Cloud to Edge webinar', 'eventValue' : undefined });" >Watch Cloud to Edge webinar</a></p>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip is-deep is-bordered">
+  <div class="row p-divider">
+    <div class="col-4 p-divider__block">
+      <h2 class="p-heading--three">Cost-saving automation</h2>
+      <p>Cut operating costs with automated scaling. You can deploy thousands of identical sites, creating an infrastructure that's built from the ground up for operations at the lowest possible cost. Canonical&rsquo;s open source tooling and focus on automation makes it easy to deploy cloud environments, again and again. </p>
+    </div>
+    <div class="col-4 p-divider__block">
+      <h2 class="p-heading--three">Carrier-grade uptime</h2>
+      <p>Deploy a carrier grade product suite in line with ETSI frameworks, while remaining aligned with upstream projects to ensure easy upgrades. Unrivalled security and resilience are intrinsic to our Communication Service Provider (CSP) solutions &mdash; which means guaranteed uptime for you.</p>
+    </div>
+    <div class="col-4 p-divider__block">
+      <h2 class="p-heading--three">Expert knowledge and experience</h2>
+      <p>Our team of specialists operates production CSP environments around the globe, covering the entire environment life-cycle, from deployment to upgrades and onboarding virtual network functions &mdash; enabling you to realise the true potential of your cloud.</p>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip is-bordered">
+  <div class="row">
+    <h2>Why choose Canonical?</h2>
+    <ul class="p-list is-split">
+      <li class="p-list__item is-ticked">Keep CAPEX and OPEX costs down with virtualisation and automation</li>
```

Comment:
Could you add abbreviations here please?

Reply:
Done